iaso/api/mobile/entity_type.py

```diff
@@ -0,0 +1,99 @@
+from django.core.exceptions import ValidationError
+from django.db.models import Prefetch
+from django_filters.rest_framework import DjangoFilterBackend  # type: ignore
+from rest_framework import filters, status
+from rest_framework import serializers
+from rest_framework.decorators import action
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+from iaso.api.common import (
+    ModelViewSet,
+    TimestampField,
+)
+from iaso.api.mobile.entity import MobileEntitySerializer
+from iaso.models import Entity, Instance, EntityType
+from iaso.utils.jsonlogic import jsonlogic_to_q
+
+
+class LargeResultsSetPagination(PageNumberPagination):
+    page_size = 1000
+    page_size_query_param = "page_size"
+    max_page_size = 1000
+
+
+class MobileEntityTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = EntityType
+        fields = [
+            "name",
+            "created_at",
+            "updated_at",
+            "reference_form",
+            "account",
+            "is_active",
+        ]
```

Comment:
The list of fields returned by the `MobileEntityType` serializer should be the same as the one returned by the `EntityTypeSerializer`:

- id
- name
- created_at
- updated_at
- reference_form
-  entities_count
- account
- fields_detail_info_view
- fields_list_view